uint8 command
uint8 controller_requested
uint8[100] control_data
---
uint8 status
float32 time
float32 body_angle
float32 motor_angleA
float32 motor_angleB
float32 leg_angleA
float32 leg_angleB
float32 motor_torqueA
float32 motor_torqueB
float32 xPosition
float32 yPosition
float32 zPosition
float32 xVelocity
float32 yVelocity
float32 zVelocity
float32 horizontal_velocity
uint8[256] control_state
